admin_manual/enterprise/firewall/file_firewall.rst

```diff
@@ -2,39 +2,35 @@
 File Firewall
 =============
 
-The File Firewall GUI enables you to create and manage firewall rule sets from 
-your ownCloud admin page's Security section. The File Firewall gives you finer-grained control of 
-access and sharing, with rules for allowing or denying access, and restrictions 
-per group, upload size, client devices, IP address, time of day, and many more 
-criteria. For additional flexibility the File Firewall also supports regular 
-expressions.
-
-Each rule consists of one or more conditions. A request matches a rule if 
-all conditions evaluate to true. If a request matches at least one of the 
-defined rules, the request is blocked and the file content can not be read or
-written.
+The File Firewall GUI enables you to manage firewall rule sets. You can find it in your ownCloud admin page, under ``Admin -> Security``. 
+The File Firewall lets you control access and sharing in fine detail, by creating rules for allowing or denying access restrictions based on: *group*, *upload size*, *client devices*, *IP address*, *time of day*, as well as many more criteria.
+In addition to these restriction options, the File Firewall app also supports rules based on `regular expressions`_. 
+
+How the File Firewall Works
+---------------------------
+
+Each firewall rule set consists of one or more conditions. 
+If a request matches all of the conditions, in at least one rule set, then the request is blocked by the firewall.
+Otherwise, the request is allowed by the firewall. 
 
 .. note::
-   As of ownCloud 9.0, the File Firewall app cannot lock out administrators from the Web interface when      rules are misconfigured.
+   The File Firewall app cannot lock out administrators from the web interface when rules are misconfigured.
 
-Figure 1 shows an empty firewall configuration panel. Set the logging level to 
-**Blocked Requests Only** to see blocked requests in the log. Set the logging 
-level to **All Requests** to also see allowed requests in the log for debugging. 
+Using the File Firewall
+-----------------------
 
-Create a new ruleset by clicking the **Add Group** button. 
+Figure 1 shows an empty firewall configuration panel. 
+Set your logging level to **Failures Only** for debugging, and create a new rule set by clicking the **Add Group** button. 
```

Comment:
s/Failures Only/Blocked Requests Only/
like in https://github.com/owncloud/documentation/blob/master/admin_manual/enterprise/firewall/images/firewall-2.png